Gems/Atom/RPI/Code/Include/Atom/RPI.Public/Scene.h

```diff
@@ -276,6 +276,8 @@ namespace AZ
 
             RHI::ShaderInputConstantIndex m_timeInputIndex;
             float m_simulationTime;
+            RHI::ShaderInputConstantIndex m_prevTimeInputIndex;
```

Comment:
Nit: you can use AZ::RHI::ShaderInputNameIndex which can simplify the code (remove the FindShaderInputConstantIndex call and also the IsValid() call).
If you make the change, can you update m_timeInputIndex's type as well.

Reply:
Sounds good, updated them both!